eva-accession-core/src/main/java/uk/ac/ebi/eva/accession/core/SubmittedVariantAccessioningService.java

```diff
@@ -17,18 +17,139 @@
+        return joinLists(accessioningService.getByAccessions(accessions),
+                         accessioningServiceDbsnp.getByAccessions(accessions));
+    }
+
+    @Override
+    public AccessionWrapper<ISubmittedVariant, String, Long> getByAccessionAndVersion(Long accession, int version)
+            throws AccessionDoesNotExistException, AccessionMergedException, AccessionDeprecatedException {
+        if (accession >= accessioningMonotonicInitSs) {
+            return accessioningService.getByAccessionAndVersion(accession, version);
+        } else {
+            return accessioningServiceDbsnp.getByAccessionAndVersion(accession, version);
+        }
+    }
+
+    @Override
+    public AccessionVersionsWrapper<ISubmittedVariant, String, Long> update(Long accession, int version,
+                                                                            ISubmittedVariant iSubmittedVariant)
+            throws AccessionDoesNotExistException, HashAlreadyExistsException, AccessionDeprecatedException,
+            AccessionMergedException {
+        if (accession >= accessioningMonotonicInitSs) {
+            return accessioningService.update(accession, version, iSubmittedVariant);
+        } else {
+            return accessioningServiceDbsnp.update(accession, version, iSubmittedVariant);
+        }
+    }
+
+    @Override
+    public AccessionVersionsWrapper<ISubmittedVariant, String, Long> patch(Long accession, ISubmittedVariant variant)
+            throws AccessionDoesNotExistException, HashAlreadyExistsException, AccessionDeprecatedException,
+            AccessionMergedException {
+        if (accession >= accessioningMonotonicInitSs) {
+            return accessioningService.patch(accession, variant);
+        } else {
+            return accessioningServiceDbsnp.patch(accession, variant);
+        }
+    }
+
+    @Override
+    public void deprecate(Long accession, String reason)
+            throws AccessionMergedException, AccessionDoesNotExistException, AccessionDeprecatedException {
+        if (accession >= accessioningMonotonicInitSs) {
+            accessioningService.deprecate(accession, reason);
+        } else {
+            accessioningServiceDbsnp.deprecate(accession, reason);
+        }
+    }
+
+    @Override
+    public void merge(Long accession, Long accession1, String reason)
+            throws AccessionMergedException, AccessionDoesNotExistException, AccessionDeprecatedException {
+        if (accession >= accessioningMonotonicInitSs && accession1 >= accessioningMonotonicInitSs) {
+            accessioningService.merge(accession, accession1, reason);
+        } else if (accession < accessioningMonotonicInitSs && accession1 < accessioningMonotonicInitSs) {
+            accessioningServiceDbsnp.merge(accession, accession1, reason);
+        } else {
+//          TODO: Support merging submittedVariants and DbsnpSubmittedVariants
```

Comment:
Explain why it's not done. is it a design decision? is it done in a posterior commit? will it be done in another branch? If it's meant to be like this, keep only the exception

Reply:
According to an offline talk with @cyenyxe it will be implemented in a separate ticket [EVA-1301](https://www.ebi.ac.uk/panda/jira/browse/EVA-1301)